cmd/launcher/svc_windows.go

```diff
@@ -232,3 +249,75 @@ func (w *winSvc) Execute(args []string, r <-chan svc.ChangeRequest, changes chan
 		}
 	}
 }
+
+// determineRootDirectory is used specifically for windows deployments to override the
+// configured root directory if another one containing a launcher DB already exists
+func determineRootDirectory(slogger *slog.Logger, opts *launcher.Options) string {
+	optsRootDirectory := opts.RootDirectory
+	// don't mess with the path if this installation isn't pointing to a kolide server URL
+	if opts.KolideServerURL != "k2device.kolide.com" && opts.KolideServerURL != "k2device-preprod.kolide.com" {
```

Comment:
For my own understanding -- is the point of doing this over checking`opts.KolideHosted` to preserve dev/test environments?

Reply:
yes, when I was looking at that logic it seemed like we set the KolideHosted for localhost (and other) cases so I just kept the check to this

Reply:
I think we haven't really committed to `opts.KolideHosted` and I don't even know if it's wholly working. (but yeah, also need to carveout dev)